### DISCUSSION TITLE
the criteria for automatic allocation

### DISCUSSION THREAD

**Discussion by jihee-k** [chatwoot/chatwoot#9637] [Q&A]
Does anyone know the criteria for automatic allocation?
I feel like there are 3 people in my team, but one day I get allocated to "a" and one day I get assigned to "b" especially a lot.

**Comment by sojan-official** [comment]
Chatwoot assigns conversations in the round-robin between the online agents. 
If there is a case where only one or two agents are online and the other one isn't, then the majority of the conversations will be assigned to the online agents

ref: https://www.chatwoot.com/hc/user-guide/articles/1677696868-assigning-conversations-in-a-round_robin-fashion

**Reply by jihee-k** [reply]
@sojan-official 

Thank you for your reply. As you said, I've registered all the agents there

But I felt that the chat was assigned a lot to a person named a yesterday and a lot to a person named b the next day, so is there any standard for assignment?

Is it just automatic?
